backend/tournesol/serializers.py

```diff
@@ -324,3 +324,12 @@ def validate(self, attrs):
 
 class ContributorRatingUpdateAllSerializer(Serializer):
     is_public = BooleanField()
+
+
+class StatisticsSerializer(Serializer):
+    user_count = IntegerField()
+    last_month_user_count = IntegerField()
+    video_count = IntegerField()
+    last_month_video_count = IntegerField()
+    comparison_count = IntegerField()
+    last_month_comparison_count = IntegerField()
```

Comment:
This serializer looks great, but I think it can look ever better : ]

If we want to add more stats in the future, for instance, the total number of users in addition to the total number of active users, we may want to reword some fields.

Also, the field `user_count` can be confusing as we are not returning the number of users, but the number of active users.

In order to keep the API as stable as possible, I'd suggest reword it now to keep " doors open ".

What do you think about this suggestion @farnots , @lfaucon and @amatissart ?

```python
class StatisticsSerializer(Serializer):
    users_active = IntegerField()
    users_active_last_month = IntegerField()
    videos_compared = IntegerField()
    videos_compared_last_month = IntegerField()
    comparisons = IntegerField()
    comparisons_last_month = IntegerField()
```

I like to use the model name as prefix (`users_`, `videos_`), as it allows to visually regroup statistics and allows to quickly identify which object is counted. The drawback is field names that are not correct English nouns.

As it's a personal preference I didn't use the GitHub suggestion feature.

Reply:
Another solution would be to split the statistics into 3 objects: `active_users`, `compared_videos` and `comparisons`.
But this API is not critical and can be extended or changed with little difficulty, so any solution is reasonable in the short term.

In the longer term, we could wait for the specifications in #509, and define a more generic schema with statistics per ballot.  
Something like:  
```json
{
  "active_users": {
    "total": 123,
    "joined_last_month": 42
  },
  "ballots": [
    {
      "name": "youtube",
      "compared_entities": {
        "total": 1234,
        "added_last_month": 234
      },
      "comparisons": {
        "total": 12345,
        "edited_last_month": 2345
      }
    }
  ]
}
```

Reply:
I like this a lot!

As we are going to rework this endpoint after the " ballot " update, I propose to keep the serializer as is for now, and merge it to avoid having many blocked pull requests.

Reply:
I like your suggestion @GresilleSiffle about the wording. Do you want me to update it now or I wait for a complete refactor when the ballot update will be merged ?

Reply:
Let's keep it as it for now :+1: